slices/libgomp1.yaml

```
@@ -0,0 +1,9 @@
+package: libgomp1
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    # Although gcc-12-base is a package dependency, it is not needed for this libs slice and is thus omitted.
+    contents:
+      /usr/lib/*-linux-gnu/libgomp.so.*:
```

```suggestion
      /usr/lib/*-linux-*/libgomp.so.1*:
```

`gnu` need to be replaced by `*` otherwise it won't work for `arm`